site_scons/prereq_tools/base.py

```diff
@@ -46,6 +46,7 @@
 from SCons.Script import SetOption
 from SCons.Script import Configure
 from SCons.Script import AddOption
+from SCons.Script import WhereIs
```

Comment:
(pylint-no-name-in-module) No name 'Script' in module 'SCons'
(pylint-syntax-error) Cannot import 'SCons.Script' due to syntax error 'invalid syntax (<unknown>, line 58)'